platform/ios/src/MGLMapView.h

```diff
@@ -75,6 +75,15 @@ typedef NS_ENUM(NSUInteger, MGLAnnotationVerticalAlignment) {
  
  @note You are responsible for getting permission to use the map data and for
  ensuring that your use adheres to the relevant terms of use.
+ @note Adding your own gestures to MGLMapView will block the default gesture recognizer
+ built into MGLMapView.  If you would like to use your own, you will need to implement 
+ `gestureRecognizer:shouldRecognizeSimultaneouslyWithGestureRecognizer:` in order to setup
```

Comment:
Qualify this method fully as `-[UIGestureRecognizerDelegate gestureRecognizer:shouldRecognizeSimultaneouslyWithGestureRecognizer:]`. Eventually jazzy will autolink Apple APIs like this one: realm/jazzy#13.